frontend/src/components/starmap/StarMap.vue

```diff
@@ -274,22 +269,22 @@ const onCanvasClick = (event: MouseEvent) => {
 
 const onCanvasMouseMove = (event: MouseEvent) => {
   const star = getRaycasterIntersects(event)
-  if (star) {
-    // Get the position of the star on the screen
-    const vector = new Vector3(...cartesianFromSphere(star.azimuth, star.altitude, star.distance))
-    const canvas = renderer.domElement
-    vector.project(camera)
-
-    const x = Math.round((0.5 + vector.x / 2) * (canvas.width / window.devicePixelRatio))
-    const y = Math.round((0.5 - vector.y / 2) * (canvas.height / window.devicePixelRatio))
-
-    // Set data for info component
-    hoveredStar.x = x
-    hoveredStar.y = y
-    hoveredStar.data = star.data
-  } else {
+  if (!star) {
     hoveredStar.data = null
+    return
   }
+  // Get the position of the star on the screen
+  const vector = new Vector3(...cartesianFromSphere(star.azimuth, star.altitude, star.distance))
+  const canvas = renderer.domElement
+  vector.project(camera)
+
+  const x = Math.round((0.5 + vector.x / 2) * (canvas.width / window.devicePixelRatio))
+  const y = Math.round((0.5 - vector.y / 2) * (canvas.height / window.devicePixelRatio))
+
+  // Set data for info component
+  hoveredStar.x = x
+  hoveredStar.y = y
+  hoveredStar.data = star.data
```

Comment:
```suggestion
  const { data } = star
  Object.assign(hoveredStart, { x, y, data })
```

Nitpicking obviously, do as you please.